composer.json

```diff
@@ -19,9 +19,9 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder":           "~2.3|~3.0",
-        "symfony/validator":        "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.1",
```

Comment:
@purposefull I am not quite sure if we on the same track. `~3.0` version doesn't prevent you from installing the package on 3.1 or 3.20. Please, check out https://getcomposer.org/doc/articles/versions.md#tilde.

@andreybolonin why should we limit 3.0 users? This bundle is still compatible with 3.0. Moreover, symfony 3.0 is supported yet,  please, check out http://symfony.com/roadmap?version=3.0#checker